migrate/20230725_billing_meters.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:billing_meter) do
```

Comment:
Isn't it more of a `billing_rate`?

Reply:
Naming comes from [Azure](https://learn.microsoft.com/en-us/rest/api/containerapps/stable/billing-meters/get?tabs=HTTP#billingmeter), and it seems it is used in [other places](https://www.moesif.com/docs/guides/guide-on-creating-a-billing-meter-with-stripe/) as well, but I don't have strong attachment to this name. `billing_rate` is also good and possibly more meaningful to people who are not familiar with Azure.